indispensable-josef/src/commonMain/kotlin/at/asitplus/signum/indispensable/josef/JsonWebKey.kt

```diff
@@ -301,15 +303,20 @@ data class JsonWebKey(
             }
 
             JwkType.RSA -> {
-                CryptoPublicKey.RSA(
+                RSA(
```

Comment:
please revert this, it is explicit for a reason (to avoid `JwkType.RSA` and `CryptoPublicKey.RSA` being confusing)

Reply:
noted

Reply:
done